manifests/supervisorcluster/1.20/cns-csi.yaml

```diff
@@ -251,6 +256,10 @@ spec:
               value: "50"
             - name: INCLUSTER_CLIENT_BURST
               value: "50"
+            - name: VSPHERE_CLOUD_OPERATOR_SERVICE_NAME
```

Comment:
I believe this is not being used anywhere. We only need this in the external provisioner.